index.js

```diff
@@ -78,6 +80,8 @@ class GoogleAuth extends DbBase {
         blockPrivilege TINYINTEGER,
         analyticsPrivilege TINYINTEGER,
         manageAdminsPrivilege TINYINTEGER,
+        passwordResetToken TEXT,
```

Comment:
add a migration for these fields in the migration file - https://github.com/bitfinexcom/bfx-facs-auth-google/blob/master/migrations.js